test/spec/modules/identityLinkIdSystem_spec.js

```diff
@@ -124,4 +129,29 @@ describe('IdentityLinkId tests', function () {
     );
     expect(callBackSpy.calledOnce).to.be.true;
   });
+
+  it('should not call the LiveRamp envelope endpoint if cookie _lr_retry_request exist', function () {
+    let now = new Date();
+    now.setTime(now.getTime() + 3000);
+    storage.setCookie('_lr_retry_request', 'true', now.toUTCString());
+    let callBackSpy = sinon.spy();
+    let submoduleCallback = identityLinkSubmodule.getId(defaultConfigParams).callback;
+    submoduleCallback(callBackSpy);
+    let request = server.requests[0];
+    expect(request).to.be.eq(undefined);
+  });
+
+  it('should call the LiveRamp envelope endpoint if cookie _lr_retry_request does exist', function () {
```

Comment:
Is the wording for this unit test correct? It looks similar to the test just above it. Should it not read, "should call the LiveRamp envelope endpoint if cookie _lr_retry_request does NOT exist" ?

Reply:
Yeah, you are right...I will change it right away. Tnx